src/class8FileIO/HomeworkFileIO.java

```diff
@@ -0,0 +1,175 @@
+
+    // 5. Write a Java program to get file size in bytes, kb, mb.
+    public int getFileSize(String directory, String sizeFormat) throws FileNotFoundException, IllegalArgumentException {
+        File file = new File(directory);
+        int fileSize;
+        int fileSizeBytes = (int) file.length();
+        int fileSizeKB = fileSizeBytes / 1024;
+        int fileSizeMB = fileSizeKB / 1024;
+        if (!file.exists()) {
+            throw new FileNotFoundException(NO_FILE_IN_DIRECTORY.getValue());
+        } else {
+            switch (sizeFormat) {
+                case "bytes":
+                    fileSize = fileSizeBytes;
+                    break;
+                case "kb":
+                    fileSize = fileSizeKB;
+                    break;
+                case "mb":
+                    fileSize = fileSizeMB;
+                    break;
+                default:
+                    throw new IllegalArgumentException(INVALID_FILE_SIZE.getValue());
+            }
+        }
+        return fileSize;
+    }
+
+    // 6. Write a Java program to store text file content line by line into an array.
+    public String fileContentIntoArray(String filePath) throws IOException {
+        BufferedReader in = new BufferedReader(new FileReader(filePath));
+        String str;
+        List<String> list = new ArrayList<String>();
+        while (((str = in.readLine()) != null)) {
+            list.add(str);
+        }
+        if (list.isEmpty()) {
+            throw new IOException(EMPTY_FILE.getValue());
+        }
+        return list.toString();
+    }
+
+    // 7. Write a method that will count the number of lines in each file that is specified on the parameter.
+    public int countLines(String filePath) throws IOException {
+        BufferedReader in = new BufferedReader(new FileReader(filePath));
+        int lineCount = 0;
+        while (((in.readLine()) != null)) {
+            lineCount++;
+        }
+        return lineCount;
+    }
+
+    // 8. Write a Java program to read content from one file and write it into another file.
+    public void readAndWriteContent(String readFilePath, String writeFilePath) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(readFilePath));
+             BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(writeFilePath))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                bufferedWriter.write(line);
+                bufferedWriter.newLine();
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            System.out.println();
+            e.printStackTrace();
+        }
+    }
+
+    // 9. Write a copy file and delete file methods.
+    public void copyFile(String sourcePath, String destinationPath) {
+        Path path1 = Paths.get(sourcePath);
+        Path path2 = Paths.get(destinationPath);
+        try {
+            Files.copy(path1, path2);
```

Comment:
:D esi uzum ei duq greq sra functionality-in, es methody chogtagorceiq))
bayc de eli lava, vor gtel es)) :D